src/subapps/dataExplorer/DataExplorerUtils.tsx

```diff
@@ -175,6 +243,35 @@ export const getUniquePathsForProperties = (
   return Array.from(new Set(paths));
 };
 
+export type PropertyPath = {
+  label: string;
+  value: string;
+};
+
+export const getUniquePathsForProperties2 = (
```

Comment:
can we find a better name

Reply:
:sweat_smile: yes. I can. I'll update this soon.